src/Microsoft.Sbom.Common/Constants.cs

```diff
@@ -20,4 +20,7 @@ public static class Constants
 
     public const string DefaultManifestInfoName = "SPDX";
     public const string DefaultManifestInfoVersion = "2.2";
+
```

Comment:
Is the extra newline intentional?

Reply:
Yup, just following the structure and keeping the defaults separated from the header names.